samples/README.md

```diff
@@ -1,116 +1,108 @@
-   ```
-6. Download `pubsublite-spark-sql-streaming-$CONNECTOR_VERSION-with-dependencies.jar` from [Maven Central](https://search.maven.org/artifact/com.google.cloud/pubsublite-spark-sql-streaming) and set `PUBSUBLITE_SPARK_SQL_STREAMING_JAR_LOCATION` environment variable.
-7. Create GCS bucket and upload both `pubsublite-spark-sql-streaming-$CONNECTOR_VERSION-with-dependencies.jar` and the sample jar onto GCS
-   ```sh
-   gsutil mb $BUCKET
-   gsutil cp target/pubsublite-spark-snippets-$SAMPLE_VERSION.jar $BUCKET
-   gsutil cp $PUBSUBLITE_SPARK_SQL_STREAMING_JAR_LOCATION $BUCKET
-   ```
-8. Set Dataproc region
-   ```sh
-   gcloud config set dataproc/region $REGION
-   ```
-9. Run the sample in Dataproc. This will perform word count aggregation and publish word count results to Pub/Sub Lite.
-   ```sh
-   gcloud dataproc jobs submit spark --cluster=$CLUSTER_NAME \
-      --jars=$BUCKET/pubsublite-spark-snippets-$SAMPLE_VERSION.jar,$BUCKET/pubsublite-spark-sql-streaming-$CONNECTOR_VERSION-with-dependencies.jar \
-      --class=pubsublite.spark.WordCount -- \
-      projects/$PROJECT_NUMBER/locations/$REGION-$ZONE_ID/subscriptions/$SOURCE_SUBSCRIPTION_ID \
-      projects/$PROJECT_NUMBER/locations/$REGION-$ZONE_ID/topics/$DESTINATION_TOPIC_ID
-   ```
-10. Read word count results from Pub/Sub Lite, you should see the result in console output.
-    ```sh
-    PROJECT_NUMBER=$PROJECT_NUMBER \
-    REGION=$REGION \
-    ZONE_ID=$ZONE_ID \
-    DESTINATION_SUBSCRIPTION_ID=$DESTINATION_SUBSCRIPTION_ID \
-    mvn compile exec:java -Dexec.mainClass=pubsublite.spark.ReadResults
-    ```
-
-## Cleaning up
-1. Delete Pub/Sub Lite topic and subscription.
-   ```sh
-   gcloud pubsub lite-subscriptions delete $SOURCE_SUBSCRIPTION_ID --zone=$REGION-$ZONE_ID
-   gcloud pubsub lite-topics delete $SOURCE_TOPIC_ID --zone=$REGION-$ZONE_ID
-   gcloud pubsub lite-subscriptions delete $DESTINATION_SUBSCRIPTION_ID --zone=$REGION-$ZONE_ID
-   gcloud pubsub lite-topics delete $DESTINATION_TOPIC_ID --zone=$REGION-$ZONE_ID
-   ```
-2. Delete GCS bucket.
-   ```sh
-   gsutil -m rm -rf $BUCKET
-   ```
-3. Delete Dataproc cluster.
-   ```sh
-   gcloud dataproc clusters delete $CLUSTER_NAME --region=$REGION
-   ```
+### Cleaning up
+```sh
+word_count_sample.sh clean
+```
+
+## Simple Read Sample
+
+### Environment Variables
+Set the following environment variables. <br>
+```
+export PROJECT_NUMBER=12345 # or your project number
+export REGION=us-central1 # or your region
+export ZONE_ID=b # or your zone id
+export SOURCE_TOPIC_ID=test-topic # or your topic id to create
+export SOURCE_SUBSCRIPTION_ID=test-subscription # or your subscription to create
+export CLUSTER_NAME=waprin-spark7 # or your Dataproc cluster name to create
+export BUCKET=gs://your-gcs-bucket
+export CONNECTOR_VERSION= # latest pubsublite-spark-sql-streaming release version
+```
+
+### Running simple read sample
+
+To run the simple read sample in Dataproc cluster:
+```sh
+simple_read_sample.sh run
```

Comment:
I'm -1 on bash scripts. They aren't windows friendly, and even on linux I had the idea of having to read a bash script before I run it.  I'd rather just have the commands in the readme so I can understand what's happening.

Reply:
I did this in an attempt to make it easier for user to run the sample, otherwise it would be like 10 steps for each sample and we have 3 samples so the readme will be very long. If the user is interested in what it actually does, they could go into the bash script and it's well commented. Open to suggestions as well, cc @anguillanneuf

Reply:
Not windows friendly is a good point. But the steps are indeed getting very long since multiple products (Dataproc, GCS, Pub/Sub Lite) are involved. I like the comments in the shell script that describe what they do. How about we briefly describe what each shell scripts does in the README, and suggest that it may be a good idea for windows users to run the commands in those script one-by-one to achieve the same result.

Reply:
Are you documenting this on Cloud Docs as well? maybe it's better to just replace the README with a reference to the Cloud Docs site instead. 

It seems like if you're at the point where you are enumerating different steps or listing separate commands for windows, you aren't saving much space and should just consolidate and list all the commands.

Reply:
We plan to have a quickstart on Cloud Docs, but that won't happen right away.

No, we aren't creating separate commands for Windows users but asking them to reference the bash scripts. I was suggesting we say something like "use this script to create a Dataproc cluster and submit a job on the cluster". 

But I see your point, I would also appreciate it if I can see the actual gcloud command for submitting a job, since there are different params I can play with. 

The bash scripts are nice for quick testing though, can we keep both the bash scripts and the long-form readme?

Reply:
I see the point of different operating system. I am going to keep the long form readme, plus adding one sentence comments about the fact that the bash scripts exist if they want to use.